crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -551,15 +558,9 @@ pub struct AuthFromJwt {
 impl AuthFromJwt {
     #[instrument(level = "debug", ret(Debug))]
     fn identity(&self, claims: &JwtClaims) -> Option<AuthId> {
-        use common::prov::AgentId;
-        use serde_json::Value;
-
-        if let Some(Value::String(external_id)) =
-            Value::Object(claims.0.clone()).pointer(&self.json_pointer)
-        {
-            Some(AuthId::agent(&AgentId::from_external_id(external_id)))
-        } else {
-            None
+        match AuthId::from_jwt_claims(claims, &self.json_pointer) {
```

Comment:
is this simply `AuthId::from_jwt_claims(claims, &self.json_pointer).ok()` or somesuch?

Reply:
What do you mean, sorry?

Reply:
Actually, I like it, I've changed it. Thanks